src/_P043_ClkOutput.ino

```diff
@@ -112,12 +149,18 @@ boolean Plugin_043(byte function, struct EventStruct *event, String& string)
           if (matchClockEvent(clockEvent,clockSet))
           {
             byte state = ExtraTaskSettings.TaskDevicePluginConfig[x];
-            if (state != 0)
+            if (state != 0) 
             {
-              state--;
-              pinMode(CONFIG_PIN1, OUTPUT);
-              digitalWrite(CONFIG_PIN1, state);
-              UserVar[event->BaseVarIndex] = state;
+              if (CONFIG_PIN1 >= 0) { // if GPIO is specified, use the old behavior
+                state--;
+                pinMode(CONFIG_PIN1, OUTPUT);
+                digitalWrite(CONFIG_PIN1, state);
+                UserVar[event->BaseVarIndex] = state;
+              }
+              else {
+                UserVar[event->BaseVarIndex] = x+1;
+                UserVar[event->BaseVarIndex+1] = state;
```

Comment:
Why is the order of task values changed if a GPIO is specified?
Also `Output_Data_type_t::Simple` makes it possible to set 3 or 4 output values, so what happens then?